### DISCUSSION TITLE
Make Links Take URL Params To Preselect Items

### DISCUSSION THREAD

**Discussion by garrettcorn** [BedrockTweaks/Files#345] [Ideas]
For example - the following link: https://bedrocktweaks.net/resource-packs?select=Alternate%20Block%20Destruction,Solid%20Honey would open the resource-packs page with Alternate Block Destruction and Solid Honey already selected.

This would make it super easy to modify packs. A user could simply save off a url after they select all of the tweaks they want and open the link in the future to add/remove items instead of manually reselecting everything each time.

**Comment by yzabeast1** [comment]
a similar feature exists on becomtweaks where you can upload an existing pack and it will auto select the ones you had selected in the pack you uploaded

**Reply by NSPC911** [reply]
I think I might add the URL Params thing to BEComTweaks as well, but only for resource packs
and if you don't know what BEComTweaks is, ~~you live under a rock~~, refer to https://becomtweaks.github.io or https://github.com/BEComTweaks
